mod_reforged/hooks/items/weapons/greenskins/orc_javelin.nut

```diff
@@ -5,4 +5,13 @@
 		create();
 		this.m.Reach = 0;
 	}
+
+	o.onEquip = function()
+	{
+		this.weapon.onEquip();
+
+		this.addSkill(::MSU.new("scripts/skills/actives/throw_javelin", function(o) {
```

Comment:
Going off of the weapon's stamina modifier, the `orc_javelin` is heavier than the regular javelin. So I think don't reduce the costs of the skills for it.

Reply:
As the barbarian javelin, this needs to be base.